sections/faqs/extend-and-styled-difference.js

```diff
@@ -0,0 +1,25 @@
+import md from 'components/md'
+
+const ExtendAndStyled = () => md`
+  ## What is the difference between \`Comp.extend\` and \`styled(Comp)\`?
+
+  ### Behind the scenes
+  The \`styled()\` factory generates new component styles with a new class. The classnames are
+  then passed to the React component via the \`className\` prop.
+  Calling \`extend\` creates new component styles by extending the old one, and thus doesn't
+  generate two classes for a single component. (\`styled()\` factory does that)
+
+  ### Which one should you use
+  We recommend to use \`.extend\` where possible for internal \`StyledComponents\`,
+  but the \`styled()\` factory for external components and cross-file containers/components
+  where it isn't clear what it's going to be when writing the wrapper.
```

Comment:
This applied to both so the combination of the two above still makes sense.

~wrapper~ `StyledComponent`

Reply:
Is `StyledLink` a styled component here?
```
import Link from 'router/link'

const StyledLink = styled(Link)``